doc/features.mld

```diff
@@ -20,15 +20,15 @@ mechanism is only used for modules and is a naming convention. If any module has
 Occasionally it’s useful to declare an item in one place and expose
 it in the public interface in another. In order to prevent unwanted occurrences
 of the actual definition site, [odoc] has a feature whereby the 'canonical'
-location of a module, module type or type can be specified.
+location of a module, module type, or type can be specified.
 
-The biggest user of module aliases is Dune's {{:https://dune.readthedocs.io/en/stable/dune-files.html#library}wrapped libraries}.
+The biggest user of module aliases is Dune's {{:https://dune.readthedocs.io/en/stable/reference/dune/library.html#library}wrapped libraries}.
 This feature allows Dune to produce a library whose interface is exposed entirely
 though a single top-level module. It does this by mangling the names of the
-real implementation modules and generating the single top-level module that
+real implementation modules. Then it generates the single top-level module that
```

Comment:
Changing this to two sentences with a different tense makes it look like it has finished "doing so" after "mangling the names". Also, there is no notion of "things being done one after the other" that the word "then" implies!

```suggestion
real implementation modules ; and generating the single top-level module that
```

Reply:
Thank you for the clarification!